### DISCUSSION TITLE
upscale like the example is impossible

### DISCUSSION THREAD

**Discussion by allbut0ne** [upscayl/upscayl#634] [Q&A]
I've tested upscayl (on my mac). It does a pretty good job!
But: the example as shown on the homepage https://www.upscayl.org/ is impossible. I've downloaded the images and tested it.
https://www.upscayl.org/low.jpg
https://www.upscayl.org/high.jpg

**Comment by aaronliu0130** [comment]
Using Ultrasharp, I get extermely similar results, but the trees are a blur.
@NayamAmarshe Maybe we should replace the "high" image?

**Comment by NayamAmarshe** [comment]
Duplicate of: https://github.com/upscayl/upscayl.github.io/issues/9

TLDR: Marketing

**Reply by aaronliu0130** [reply]
Same thing with the downloads count?
